src/config.c

```diff
@@ -2438,7 +2438,6 @@ standardConfig configs[] = {
     createBoolConfig("lazyfree-lazy-user-flush", NULL, MODIFIABLE_CONFIG, server.lazyfree_lazy_user_flush , 0, NULL, NULL),
     createBoolConfig("repl-disable-tcp-nodelay", NULL, MODIFIABLE_CONFIG, server.repl_disable_tcp_nodelay, 0, NULL, NULL),
     createBoolConfig("repl-diskless-sync", NULL, MODIFIABLE_CONFIG, server.repl_diskless_sync, 0, NULL, NULL),
-    createBoolConfig("gopher-enabled", NULL, MODIFIABLE_CONFIG, server.gopher_enabled, 0, NULL, NULL),
```

Comment:
maybe we want to skip this config in the config file parsing, like we do for `list-max-ziplist-value`, so that we don't fail starting up?
i suppose not, because i imagine no one uses it..

Reply:
In the tension between simplicity and breaking backwards compatibility I always tend towards simplicity. So if you ask me we shouldn't keep this. Also anyone having this enabled should fail startup so they won't assume this is on when we actually killed the feature. The only problem is if someone explicitly turned this off in the config file, but I doubt anyone has.

Reply:
yeah, that's what i meant, ignoring a `no` value, and aborting on `yes`.
it would be just one hunk of code, but i tend to agree that the chance it'll really do good for someone is rather low.
let's see if anyone else thinks differently.

Reply:
I would rather be explicit here and break if someone is expecting gopher to be enabled and working. I view the ziplist ones differently since they don't change functional behavior, just performance. We should add it to the release notes that it was deprecated.